python/pyspark/cloudpickle.py

```diff
@@ -1010,11 +1006,7 @@ def save_ellipsis(self, obj):
     def save_not_implemented(self, obj):
         self.save_reduce(_gen_not_implemented, ())
 
-    try:               # Python 2
-        dispatch[file] = save_file
-    except NameError:  # Python 3  # pragma: no branch
-        dispatch[io.TextIOWrapper] = save_file
-
```

Comment:
Ah, let's match cloudpickle with the released version. That's why I didnt touch this code.

Reply:
Good point.